plugins/completion/nvim-cmp/default.nix

```diff
@@ -607,38 +607,34 @@ in {
 
       inherit (cfg) view;
       window = helpers.ifNonNull' cfg.window {
-        completion = helpers.ifNonNull' cfg.window.completion {
-          inherit
-            (cfg.window.completion)
-            border
-            winhighlight
-            zindex
-            scrolloff
-            scrollbar
-            ;
-          col_offset = cfg.window.completion.colOffset;
-          side_padding = cfg.window.completion.sidePadding;
-        };
-        documentation = helpers.ifNonNull' cfg.window.completion {
-          inherit
-            (cfg.window.completion)
-            border
-            winhighlight
-            zindex
-            ;
-          max_width = let
-            inherit (cfg.window.documentation) maxWidth;
-          in
-            if isInt maxWidth
-            then maxWidth
-            else helpers.ifNonNull' maxWidth (helpers.mkRaw maxWidth);
-          max_height = let
-            inherit (cfg.window.documentation) maxHeight;
-          in
-            if isInt maxHeight
-            then maxHeight
-            else helpers.ifNonNull' maxHeight (helpers.mkRaw maxHeight);
-        };
```

Comment:
> What was the issue? Can't really understand what changed here

Indeed, the indenting changed due to formatting.
Actually the problem was in this section and each `cfg.window.completion` should have been `cfg.window.documentation`. So basically it was a copy-paste error.

Reply:
Ah, got it, will mark it as resolved then :)